pkg/platform/model/vcs.go

```diff
@@ -64,8 +64,11 @@ const (
 	// NamespaceLanguageMatch is the namespace used for language requirements
 	NamespaceLanguageMatch = `^language$`
 
-	// NamespacePackageMatch is the namespace used for package requirements
-	NamespacePackageMatch = `^language\/\w+$`
+	// NamespaceLanguagePackageMatch is the namespace used for language package requirements
+	NamespaceLanguagePackageMatch = `^language\/\w+$`
```

Comment:
Please undo. This is matching a package namespace, adding language to the name just obscures things